### DISCUSSION TITLE
Ordernumbers not found when Searching

### DISCUSSION THREAD

**Discussion by empx92** [kimai/kimai#4296] [General]
Hello,
currently we had a "problem" with the oder or/and customer numbers.
In the export section it isn't possible to search for order or customer numbers, its well known that those numbers could be searched in the "customer" or "project" field which work fine. 
But this only works as long as a project doesn't need to be split in two customers.

For example:
There is one Order with the number "2350" but the invoice goes to Customer A - Project 2350 and Customer B - Project 2350 for some reasons, but internal in our System it is the same order to get an idea of the overall costs and so on.

**Comment by kevinpapst** [comment]
The freetext search field in the invoice section does only look into the timesheet descriptions.
Maybe you want to use tags for that specific case?

Even though I do not understand what you want to accomplish, you cannot create one invoice for two customers anyway.
And if you have Project 2350 for two distinct customers, why do you want to search for it?
Just create create two invoices. 
First one for Customer A / Project 2350 
Second one for Customer B / Project 2350

**Reply by empx92** [reply]
In our case we have 3 companies (one mother and 2 "daugthers").
So there is a customer which gives us a contract for the project and internal our two companies has to work on the same project.
It's like company 1 is a general contractor. So our hierarchy would be as described above.
The Projectnumber is the Number of our Invoiceprogram(ERP) (so we do not use the inbuilt invoice as we have about 4000 Articles which are price serviced once a week by the distributor).

So when it comes to the final invoice our secretary would search for the Projectnumber within the timesheet export (which is already given as it is generated when sending the contract to the customer or writing the order, delivery note or the first offer by our ERP), which doesn't work at the moment.
So the "Projectnumber" is obsolete to us and we have to type it in to the project name which isn't really beautiful ;)
As said im not talking about the invoice section i am talking about the timesheets export.

Best regards

**Reply by kevinpapst** [reply]
I mixed invoice and export you are right.

And thanks for sharing your use-case.
Unfortunately I don't see the point 🤷  what are you trying to tell me?

**Reply by empx92** [reply]
I'm trying to tell you that we can not search for the "Projectnumbers (=Bestellnummer)" when searching in export, although you can config the calendar to show "Project - Projecnumber" which is a bit confusing.
Also i think that if using those Projectnumbers it makes sense to find it when typing in the search field on the export section.
